src/components/CreateAccount.js

```diff
@@ -3,9 +3,12 @@ import React, { Component } from 'react';
 import {
```

Comment:
add new line spacing between line 2 and 3 to separate external packages and internal